.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,44 @@
+name: 🐛 Bug report
+description: Create a bug report
+labels: [ ":bug: Type: Bug" ]
+body:
+  - type: textarea
+    attributes:
+      label: Summary
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Version
+      description: Make sure the bug is still happening on the latest version.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce the behavior
+      description: |
+        Add context about the problem here. How it happened and how to reproduce it:
+        - If your project is open source, a link to the project is greatly appreciated.
+        - If not, please share as much information as possible to help to understand the problem: schema, queries, sample code, etc...
+        - Add things that you already tried.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Logs
+      description: |
+        Paste logs and full stacktrace here.
+        You can also attach files by dragging them into the area.
+      value: |
+        ```
+        (Your logs here)
+        ```
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Anything else?
```

Comment:
This wasn't there in the original md one, but thought it could be useful

Reply:
👍